app/economy/tx.py

```diff
@@ -294,19 +482,26 @@ def get_token_originators(to_address, token, from_address='', return_what='trans
     endpoint = 'token-transfers' if token != '0x0' else 'ether-transfers'
     url = f'https://api.aleth.io/v1/{endpoint}?filter[to]=' + address + '&filter[token]=' + token + '&page%5Blimit%5D=100'
     if token == '0x0':
+        print('qqqq')
         url = f'https://api.aleth.io/v1/{endpoint}?filter[account]=' + address + '&page%5Blimit%5D=100'
     if from_address:
+        print('wwww')
```

Comment:
do we need the prints?

Reply:
Oops, yea those were leftover from my expert debugging techniques. Removed in f61690e24012169ba8d24a85fb99dc21edf26e24